### ISSUE TITLE
Performance not so good

### ISSUE THREAD

**Issue by fire717** [titu1994/keras-efficientnets#25]
Thx for this nice work.

However, this code and pretrain weights perform a little bad in my dataset.

The paper said the B1 can get same acc as xception, but in my dataset, even B4 perform much worse than xception.
(B0-B5 are all worse than xception, and I have tried change learning rate.)
(e.g. xception can get val acc 0.8, B4 can only get 0.3, while B1 can get 0.6.)

So, did I miss some important setting or param?

Thx again!
